defs/scenarios/mysql/mysql_router.yaml

```diff
@@ -0,0 +1,29 @@
+checks:
+  using_mysql_router:
+    apt: mysql-router
+  client_ssl_mode:
+    config:
+      handler: hotsos.core.plugins.mysql.MySQLRouterConfig
+      assertions:
+        key: client_ssl_mode
+        ops: [[eq, PREFERRED]]
+  client_ssl_cert:
+    config:
+      handler: hotsos.core.plugins.mysql.MySQLRouterConfig
+      assertions:
+        key: client_ssl_cert
+        ops: [[ne, null]]
+conclusions:
+  client_ssl_cert_problem:
+    decision:
+      and: [ using_mysql_router, client_ssl_mode ]
+      not: client_ssl_cert
+    raises:
+      type: MySQLWarning
```

Comment:
Ok so I should have spotted this in my last review but for the reason I have just raised in https://github.com/canonical/hotsos/issues/395 the type here needs to be "LaunchpadBug" and this will mean that in your unit test instead of calling IssuesManager().load_issues() you need to call IssuesManager().load_bugs().